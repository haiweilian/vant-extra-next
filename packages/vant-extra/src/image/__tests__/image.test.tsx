import { nextTick } from 'vue'
import { mount } from '@vue/test-utils'
import { describe, test, expect } from 'vitest'
import { Image } from '../index'

describe('Image', () => {
  test('render', async () => {
    const wrapper = mount(Image)
    await nextTick()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
