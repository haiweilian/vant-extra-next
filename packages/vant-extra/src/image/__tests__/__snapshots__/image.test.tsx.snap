// Vitest Snapshot v1

exports[`Image > render 1`] = `
"<div class=\\"van-image\\" previewimage=\\"true\\" previewoptions=\\"[object Object]\\">
  <!---->
  <div class=\\"van-image__loading\\"><i class=\\"van-badge__wrapper van-icon van-icon-photo van-image__loading-icon\\">
      <!---->
      <!---->
      <!---->
    </i></div>
  <!---->
</div>"
`;
